File: html/old/R2/SACO R2 2010 (Solutions)/Fizzbuzz.java
import java.util.Scanner;

public class Fizzbuzz {
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Count to what number? ");
        int N = sc.nextInt();

        for (int i = 1; i <= N; ++i) {
            if (i % 15 == 0)
                System.out.println("fizzbuzz");
            else if (i % 5 == 0)
                System.out.println("buzz");
            else if (i % 3 == 0)
                System.out.println("fizz");
            else
                System.out.println(i);
        }
    }
}

File: html/old/R2/SACO R2 2010 (Solutions)/fizzbuzz.py
#!/usr/bin/env python
# Keegan Carruthers-Smith - Fizzbuzz - SACO R2 2010

N = int(raw_input('Count to what number? '))
for i in range(1, N+1):
    mul_three = i % 3 == 0
    mul_five  = i % 5 == 0
    if mul_three and mul_five:
        print 'fizzbuzz'
    elif mul_five:
        print 'buzz'
    elif mul_three:
        print 'fizz'
    else:
        print i
